MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/BoundingBox.java

```diff
@@ -0,0 +1,124 @@
+package com.google.android.libraries.motionphotoreader;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import static com.google.android.libraries.motionphotoreader.Constants.EPS;
+
+/**
+ * A BoundingBox object represents the bounding box for a quadrilateral, i.e. the box enclosing the
+ * quadrilateral with the smallest measure.
+ */
+public class BoundingBox {
+
+    private static final String TAG = "BoundingBox";
+
+    /**
+     * The extreme coordinates of the bounding box.
+     */
+    public final float xMin;
+    public final float yMin;
+    public final float xMax;
+    public final float yMax;
+
+    /**
+     * A pseudo-measurement of how well the bounding box approximates the quadrilateral. This is
+     * computed by taking the maximum distance from a vertex of the quadrilateral to a neighboring
+     * vertex of the bounding box. Thus, the error lies in the range [0, max(W, H) / 2], where
+     * W is the width of the bounding box and H is the height of the bounding box.
+     */
```

Comment:
Could you put the explanation of this field in the Javadoc of the `error` function? This comment on a private field is not visible by users of this class.